cdap-hbase-compat-0.98/pom.xml

```diff
@@ -358,6 +358,27 @@
         </plugins>
       </build>
     </profile>
+    <profile>
+      <id>skipCompatTests</id>
+      <activation>
+        <property>
+          <name>skipCompatTests</name>
+          <value>true</value>
```

Comment:
In fact, the change would be simpler if we just use property.

1. At the top level pom.xml, default the property
`<skip.hbase.compat.tests>false</skip.hbase.compat.tests>`
2. In each compat module pom.xml, no need to add new profile, just override the surefire plugin with `<skip>${skip.hbase.compat.tests}</skip>`

So to skip compat module tests, run maven with `mvn -Dskip.hbase.compat.tests=true ...`

I suggest we leave it on (default) for develop branch DUT, but disable the tests for branches.